docs/reference/src/main/sitemap/sitemap.xml

```diff
@@ -0,0 +1,143 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<urlset xmlns="http://www.sitemaps.org/schemas/sitemap/0.9">
+    <url>
+        <loc>http://docs.jboss.org/weld/reference/latest-master/en-US/html/part-1.html</loc>
```

Comment:
Yeah I thought about it shortly. I am not sure if it's doable only with some maven plugin but will check.